frontend/src/components/teacher/dashboard/AssessmentsSection.tsx

```diff
@@ -47,63 +38,50 @@ const AssessmentsSection = () => {
   }, [filteredData, currentTab]);
 
   return (
-    <Box flex="1">
-      {loading && (
-        <Center flex="1" margin="15%">
-          <LoadingState />
-        </Center>
-      )}
-      {error && (
-        <Box height="100%" mt={10}>
-          <ErrorState />
-        </Box>
-      )}
-      {!!data?.length && !error && !loading && (
-        <>
-          <Tabs
-            onChange={(index) => setCurrentTab(TEST_SESSION_STATUSES[index])}
-          >
-            <TabList border="none" gap={8}>
-              {TEST_SESSION_STATUSES.map((status) => (
-                <Tab
-                  key={status}
-                  _focus={{ background: "none" }}
-                  border="none"
-                  color="blue.100"
-                  fontWeight="bold"
-                  px={0}
-                >
-                  {titleCase(status)}
-                </Tab>
-              ))}
-            </TabList>
-            <TabPanels>
-              {TEST_SESSION_STATUSES.map((status) => (
-                <TabPanel key={status} p={0}>
-                  {sortedData?.map((session) => (
-                    <TestSessionListItem
-                      key={session.testSessionId}
-                      session={session}
-                    />
-                  ))}
-                </TabPanel>
+    <StateHandler
+      emptyState={<EmptySessionsMessage />}
+      error={error}
+      isEmpty={!data?.length}
+      loading={loading}
+    >
+      <Tabs
+        mt="-2rem"
```

Comment:
Negative margins seem to indicate to me that some component somewhere is adding padding it shouldn't. Consider changing